### PR TITLE
docs - correct max value for max_connections 6.x

### PR DESCRIPTION
**Pull request by mkiyama**
Will be backported to 6X_STABLE

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -6181,7 +6181,7 @@
           </thead>
           <tbody>
             <row>
-              <entry colname="col1">10-<i>n</i></entry>
```
Comment (gaos1) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml:6184]: This is correct for master. 
https://github.com/greenplum-db/gpdb/blob/master/src/include/postmaster/postmaster.h

but for 6.x, it is 0x7fffff.  https://github.com/greenplum-db/gpdb/blob/6X_STABLE/src/include/postmaster/postmaster.h

Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/config_params/guc-list.xml:6184]: thanks

**Comment by mkiyama** [comment]
Thanks. Corrected the value.
I should check the google links more closely.

**Comment by mkiyama** [comment]
closing this PR.
Will open a PR for the 6.x max value for max_connections.

**Pull request closed without merging**